testsuite/keycloak-errors-tests/src/test/java/io/strimzi/testsuite/oauth/auth/Common.java

```diff
@@ -144,4 +150,36 @@ static String loginWithUsernamePassword(URI tokenEndpointUri, String username, S
         }
         return token.asText();
     }
+
+    public static List<String> getKafkaLogsForError(String errString) {
```

Comment:
Maybe some Javadoc to explain what it does?

Reply:
Sure.